src/main/java/com/controledegastos/application/GerenciadorDeGastos.java

```diff
@@ -0,0 +1,127 @@
+package com.controledegastos.application;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class GerenciadorDeGastos {
+
+    static final List<Taxa> TAXA_CAMBIO = Arrays.asList(
+            new Taxa(Moeda.REAL_BRASILEIRO, 1.0, 1.0),
+            new Taxa(Moeda.DOLAR_AMERICANO, 3.74, 6.0),
+            new Taxa(Moeda.PESO_CHILENO,0.0057, 2.0),
+            new Taxa(Moeda.EURO, 4.24, 8.0)
+    );
+
+    List<Gasto> gastos = new ArrayList<>();
+    
+    ControlaGastos controlaGastos = new ControlaGastos();
+
+    public void adicionaGasto() {
+        String respostaGasto = controlaGastos.responda("Digite o produto, a categoria, o valor, o estabelecimento e a moeda do país de compra");
+        gastos.add(getGasto(respostaGasto));
+    }
+
+    public void removeGasto(){
+        controlaGastos.verLista(gastos);
+        int itemRemovido = controlaGastos.selecione("Digite o número do item que você deseja remover:");
+        gastos.remove(gastos.get(itemRemovido));
+    }
+
+    public void calculaTotalGastos(){
+        Double valorTotal= 0.0;
+        for(int i=0; i<gastos.size(); i++){
+            valorTotal =valorTotal + gastos.get(i).getValor();
+        }
+        controlaGastos.mostraInformacao("Valor total da lista de gastos: ", valorTotal);
+    }
+
+    public void calculaTotalCategoria(){
+        String categoria = mostraCategorias();
+
+        List<Gasto> gastosDaCategoria = gastos
+                .stream()
+                .filter(gasto -> gasto.getCategoria().equalsIgnoreCase(categoria))
+                .collect(Collectors.toList());
+        Double valorCategoria = 0.0;
+        for(int i=0; i< gastosDaCategoria.size(); i++) {
```

Comment:
Eu vi que vocês tão usando `.stream()` ali em cima. Um desafio, mudar esse `for` por algum método de `.stream()`

(Dica: reduce ou sum) 
(Dica 2: Podemos fazer tudo na mesma stream que vocês já tem 😄 )